docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,69 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** copies container images between hosts on a network. You can load to the remote host or from the remote host as well as in between two remote hosts.
+Note: `::` is used to specify the output file on the host or the image name depending on if you are saving or loading.
+
+**podman image scp [GLOBAL OPTIONS]**
+
+**podman image** *scp [OPTIONS] NAME[:TAG] HOSTNAME[::FILELOCATION]*
+
+**podman image** *scp [OPTIONS] HOSTNAME[::IMAGENAME]*
+
+**podman image** *scp [OPTIONS] HOSTNAME[::IMAGENAME] HOSTNAME[::]*
+
+## OPTIONS
+
+#### **--quiet**, **-q**
+
+Suppress the output
+
+#### **--help**, **-h**
+
+Print usage statement
+
+## EXAMPLES
+
+
+```
+$ podman image image scp alpine
```

Comment:
Double `image`, and I’m not sure why anyone would do this, so it’s not the best example to start with.

Reply:
fixed, that was a typo sorry

Reply:
Please remove this example or move it somewhere closer to the bottom.

Reply:
Please remove this example or move it beyond the useful ones.

(I think I might have written this review comment elsewhere? If so, I apologize for bringing it up again.)